OpenLIFUData/OpenLIFUData.py

```diff
@@ -564,7 +564,7 @@ def get_session(self, subject_id:str, session_id:str) -> "openlifu.db.session.Se
     def load_session(self, subject_id, session_id) -> None:
         # === Ensure it's okay to load a session ===
         session = self.get_session(subject_id, session_id)
-        if session.transducer.id in self.getParameterNode().loaded_transducers:
+        if session.transducer_id in self.getParameterNode().loaded_transducers:
             if not slicer.util.confirmYesNoDisplay(
                 f"Loading this session will replace the already loaded transducer with ID {session.transducer.id}. Proceed?",
```

Comment:
Currently the error message crashes when trying to reload a session.  This reference to transducer.id needs to be updated in the error message.

Reply:
Good catch! Fixed, please continue the review or let me know if approved

Reply:
Everything else looks good! You can go ahead and approve!

Reply:
Yay, thanks!